lib/services/orders.ts

```diff
@@ -207,6 +207,8 @@ export async function createOrder({ paymentMethodNonce, cart = [], customer = {}
 		if(typeof promo.maxUses === 'number' && promo.maxUses <= promoUses) throw new OrdersServiceError('Promo code no longer available', 'GONE');
 	}
 
+	// targetGuestId is actually what the user sees as a "ticket", so the 1:1 restriction remains valid per upgrade product, however
```

Comment:
this is a comment about upgrades but has not been fully implemented yet due to low demand